### ISSUE TITLE
modify_permissions could be renamed as set_atomic_permissions?

### ISSUE THREAD

**Issue by mstfdkmn** [irods/irods_client_http_api#169]
`set_permission` and `modify_permissions` operations can be used to both set and modify permissions if I understood correctly. The only difference between those two operations is that the second implements in atomic way. So, for teh sake of less confusion could this be a logical proposal?

**Comment by korydraughn** [comment]
That's true. We haven't made any decisions regarding this yet, but it will be addressed in a later release.

A couple ways we could approach this ...
- Remove `set_permission` (because `modify_permissions` covers it to an extent)
- Rename the operations so the differences are obvious
